themes/grav/templates/partials/blueprints-new-folder.html.twig

```diff
@@ -1,7 +1,7 @@
 {% set form_id = form_id ? form_id : 'blueprints' %}
 {% set scope = scope ?: 'data.' %}
 
-<form id="{{ form_id }}" method="post" data-grav-form="{{ form_id }}" data-grav-keepalive="true">
+<form action="{{ form_action }}" id="{{ form_id }}" method="post" data-grav-form="{{ form_id }}" data-grav-keepalive="true">
```

Comment:
HTML 5 specs do not allow empty action. To avoid any possible problems (though browsers do work with it), I would remove the attribute when it's not being used.

Reply:
@mahagr Sure, done!